protocols/v2/subprotocols/common-messages/src/setup_connection.rs

```diff
@@ -391,13 +406,31 @@ mod test {
     #[test]
     fn test_check_flag() {
         let protocol = crate::Protocol::MiningProtocol;
-        let flag_available = 0b_0000_0000_0000_0000_0000_0000_0000_0000;
+        let flag_avaiable = 0b_0000_0000_0000_0000_0000_0000_0000_0000;
```

Comment:
it seems we're changing this line just to introduce a typo?

Reply:
will correct it out